### PR TITLE
fix: add downloading tips and add SHA for updating github files

### PR DESCRIPTION
**Pull request by lfbdev**
# Summary
1. There is no printing while downloading 
image

2. SHA is necessary for del or update 
Before:
image

After:
image

### Related Issues
#50

### CODE REVIEW

**Comment by daniel-hutao** [comment]
image

If this pr is ready for review, please transfer it to "Ready for review":

image

**Comment by lfbdev** [comment]
@daniel-hutao plz check

File: internal/pkg/githubactions/githubactions.go [hunk 1]
```diff
@@ -109,27 +109,28 @@ func (ga *GithubActions) DeleteWorkflow(workflow *Workflow) error {
 	return nil
 }
 
-func (ga *GithubActions) fileExists(filename string) (bool, error) {
-	_, _, resp, err := ga.client.Repositories.GetContents(
+// getFileSHA: if file exists, return SHA, or return "", for file check and file delete
```
Comment (daniel-hutao) [inline internal/pkg/githubactions/githubactions.go:112]: Maybe this comment could be more clear:
1. file exists and no error occurred: return (string(SHA), nil)
2. some errors occurred: return ("", errl)
3. file not exists and no error occurred: return ("", nil)

**Review by daniel-hutao — CHANGES_REQUESTED** [review]
only one comment is added.

**Comment by daniel-hutao** [comment]
Please squash the commit history just to make the records more clear.

**Comment by daniel-hutao** [comment]
One small tip: you can click the "Resolve conversation" button so that I'll know the "review change" is down. @lfbdev

File: internal/pkg/githubactions/githubactions.go [hunk 2]
```diff
@@ -111,8 +110,8 @@ func (ga *GithubActions) DeleteWorkflow(workflow *Workflow) error {
 	return nil
 }
 
-func (ga *GithubActions) fileExists(filename string) (bool, error) {
-	_, _, resp, err := ga.client.Repositories.GetContents(
+func (ga *GithubActions) fileExists(filename string) (bool, string, error) {
```
[the remark this answers is not in the document]
Comment (lfbdev) [reply internal/pkg/githubactions/githubactions.go]: resolved

File: internal/pkg/githubactions/githubactions.go (hunk 1 above)
Comment (lfbdev) [reply internal/pkg/githubactions/githubactions.go]: function note added

**Comment by daniel-hutao** [comment]
That's all.
You can merge this pr after the commits is squashed.

/lgtm

**Pull request merged**